\documentclass[12pt]{paper}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
%\usepackage{hyperref}
\usepackage{color}
\usepackage{float}
\begin{document}
\title{Dynamic Random Loops Can Explain the Appearance of Topologically Associating Domain in Chromosome Capture Experiments}
\maketitle

\section{Introduction}\label{section_introduction}
%> The relationship between chromosomal spatio-temporal organization and chromosomal activity is incompletely understood

The spatio-temporal organization of the chromatin plays an essential role in the regulation of sub-cellular activity such as gene expression and regulation \cite{cremer2001chromosome}. 

%> Dynamic looping events between enhancer and promoters (like) are frequent in the nucleus and are related to the chromosomal dynamic organization

%> Chromosome Capture method creates loci contact maps, which are used to record chromosomal static looping events

%> Static looping events, coupled with dnamic polymer models, can shed light on the chromosoamal spatio-temporal organization 

%> previous work on the subject 
3d structure of the Igh locus was suggested by \cite{jhunjhunwala20083d} after tagging several position along the loci. 

%> our work

\section{Experimental data and Methods}\label{section_methods}

%> the experimental data from 5c experiments includes a subset of 2 TADs out of the Nora et al data
We use the experimental 5C data generated by Nora et al.\cite{Nora2012} for the chromosome contact frequencies of the X chromosome in a 4.5 Gb region encompassing the X inactivation center. In our work we focus on a subset of the data, including a 94,082 bp region, termed TAD D and E (see \cite{Nora2012}). 

%> coarse graining of the data followed the method of Giorgetti et al 
The maps generated by the 5C experiments describes the contact between genomic loci of variable sizes in many nuclei. Contacting loci are those resulted from the cross-linking created by the fixation of DNA in formaldehyde, which  yields a contact distance of 10-100 nm \cite{dekker2013exploring}. We then follow data coarse-graining procedure performed by Giorgetti et al.\cite{Giorgetti2014} to map segment encounter data to that of beads. A bead size of 3000 bp was chosen according to the mean size of restriction segments resulted by HindIII enzyme digestion, used in the process of the 5C experiments (see \cite{Giorgetti2014} Supplementary Material). This choice of bead size resulted in coarse-grained polymer of 307 beads. 

%> the calculation of the bead encounter probability 
The coarse-grained pair-wise bead encounter frequency includes 14,509 data points and was used to calculate the bead encounter probability as a function of bead distance in the following way. Encounter frequencies for equidistant beads were averaged to give the 'one sided' bead encounter frequency, we then divide the bead encounter frequency by the total number of encounters for that bead to obtain the encounter probability. We use the bead encounter probability to display the bead encounter matrix. This is a 307 by 307 matrix containing the pair-wise encounter probabilities in each of its entries.

%> Rouse polymer is used to represent the chromosom
In our simulations we use a Rouse polymer to examine different polymer conformation that give rise to the appearance of a Topologically Associating Domain (TAD). The Rouse model describes the dynamics of a linear polymer as a collection of massless beads connected by harmonic springs and driven by the thermal forces of diffusion. 
The differential equation describing the time progression of the $n^{th}$ bead in a chain of $N$ beads is given by 
\begin{equation}
\frac{dR_n}{dt}=-\frac{K_BT}{b^2\zeta}(2R_n(t)-R_{n-1}(t)-R_{n+1}(t))+\sqrt{2D}\frac{dw_n}{dt}
\end{equation}
where $R_n(t)$ describes the 3D coordinates of bead $n$ at time $t$, $K_BT/\zeta$ is the diffusion constant, $b$ is the standard deviation of the distance between adjacent beads of the chain, and $w_n$ is a white Gaussian noise. 

%> transfroming to normal coordinates

%> previous results of the MFPT (Amitai 2012)

%> the prcess of obtaining the MFET from the normal coordiantes linear chain case
%1. transfrom from spatial coordinates to normal coordinates
%2. solve the fokker planck equation (find suitable boundary conditions)
%3. express the solution as an infinite series
%4. integrate to get the CDF adn calculate the survival
%5. integrate the survival to get the MFET 



%> fitting the experimental data and comparing to the theory
The encounter probability was fitted with a function of the form  $\alpha d^{-\beta}$, with $\alpha = \frac{1}{\sum_{j=1}^k j^{-\beta}}$, $d$ is the distance in bead units, and $\beta$ is a parameter to be determined.



\section{Results}\label{section_results}
%> Analysis of the experimental data, show the mean value of beta
%> show that the data contains peaks that might correspond to hard coded big loops 
%> analyze the region containing the big loop in term of eigenvalues 
%> analyze the normal coordinate system to find the encounter probability 
%> show that the encounter probability beta value drops quadratically with the number of internal dynamic random loops 
%> show the connection between the average beta and the average number of loops 
%> argue that in the resultion of data that we have, we can only say something concrete about the large loop. 
%> try to find the connection between gene expression and the number of loops the model predicted 
%> try to estimate gene expression rate according to the encounter probability 



\section{Discussion}\label{section_discussion}
\bibliographystyle{plain}
\bibliography{randomLoopsBibliography} % the bibliography.bib
\end{document}